benchmarks/Microsoft.AspNetCore.Routing.Performance/LinkGeneration/LinkGenerationGithubBenchmark.cs

```diff
@@ -57,9 +57,10 @@ public void TreeRouter()
         [Benchmark]
         public void EndpointRouting()
         {
-            var actualUrl = _linkGenerator.GetLink(
+            var actualUrl = _linkGenerator.GetPathByRouteValues(
                 _requestContext.HttpContext,
-                values: new RouteValueDictionary(_lookUpValues));
+                routeName: null,
```

Comment:
Isn't getting a path by values alone fairly common? What do you think of adding GetPathByRouteValues overloads without name?